modules/widget/page_detail/PageDetailWidgetModule.php

```diff
@@ -138,28 +138,22 @@ public function getPageTypes() {
 	private function getAvailablePageProperties($oPage) {
 		$aAvailablePageProperties = $oPage->getTemplate()->identifiersMatching('pageProperty', Template::$ANY_VALUE);
 		$aResult = array();
-		$aSetProperties = array();
-		foreach($oPage->getPagePropertyQuery()->byNamespace(false)->find() as $oPageProperty) {
-			$aSetProperties[$oPageProperty->getName()] = $oPageProperty->getValue();
-		}
+
 		foreach($aAvailablePageProperties as $oProperty) {
 			$sPropertyName = $oProperty->getValue();
-
-			$aResult[$sPropertyName]['value'] = isset($aSetProperties[$sPropertyName]) ? $aSetProperties[$sPropertyName] : '';
+			$aResult[$sPropertyName]['value'] = '';
 			$aResult[$sPropertyName]['defaultValue'] = $oProperty->getParameter('defaultValue');
 			$aResult[$sPropertyName]['type'] = $oProperty->getParameter('propertyType');
-
-			unset($aSetProperties[$sPropertyName]);
 		}
-		foreach($aSetProperties as $sRemainingPropertyName => $sRemainingPropertyValue) {
-			$aResult[$sRemainingPropertyName] = array('value' => $sRemainingPropertyValue, 'defaultValue' => null, 'type' => null);
+		foreach($oPage->getPagePropertyQuery()->byNamespace(false)->find() as $oPageProperty) {
+			if(isset($aResult[$oPageProperty->getName()])) {
+				$aResult[$oPageProperty->getName()]['value'] = $oPageProperty->getValue();
+			}
```

Comment:
```suggestion
			} else {
				$aResult[$oPageProperty->getName()] = array('value' => $sRemainingPropertyValue, 'defaultValue' => null, 'type' => null);
			}
```

Reply:
Sonst werden die nicht angezeigt, die es nur in der DB gibt aber nicht im Template

Reply:
Ich dachte eigentlich, dass es nur page_properties mit name space gibt, die ja in php explizit werden, oder dann als Template Identifier gesetzt sein müssen. An welche Möglichkeit denkst du? Wie könnte ein DB-Eintrag erscheinen, dem nicht ein Identifier gegenübersteht? Und wenn wir den anzeigen, wo würde der verwendet?

Reply:
Zum Beispiel, wenn man das Template wechselt und das neue die bereits eingefüllten Property-Namen nicht mehr anbietet…

Reply:
Ausserdem hat der alte Code so funktioniert.